plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/PartitionFields.java

```diff
@@ -100,6 +100,7 @@ private static String toPartitionField(PartitionSpec spec, PartitionField field)
             case "month":
             case "day":
             case "hour":
+            case "void":
```

Comment:
Is there an enum for transforms in Iceberg library? If so we should use it to let errorprone figure out the missing cases for us (if new ones get added in future versions).

Reply:
there doesn't seem to be. transforms are constructed in `org.apache.iceberg.transforms.Transforms` and i didn't see a useful enum there.